### ISSUE TITLE
User Settings Page

### ISSUE THREAD

**Issue by CaptainAchilles** [hkhosrav/RiPLE-Client-side#6]
As per our discussions on 12/06/2017, we should allow the user to manage what times they are free for use with (#3). 

- [x] A calendar of some sort could work best where the user can click & drag their available times (and also set it to be tentative).
- [x] The user should be able to specify if they want to `Mentor`, `Be Mentored`, or `Find A Study Budy`
- [x] The settings page should be a single `View`, which contains all of the settings for other components. They can be organised by `Tabs`, or by simple section headings.
- [x] The settings page should be accessible through the side menu (perhaps by sub-menus)?

**Comment by solmazabdi** [comment]
I think specifying their willingness for being a mentor, be mentored and a
study budy shouldn't be in setting. It should be in "Connect with Peers"
tab.

**Comment by hkhosrav** [comment]
I think there are pros and cons to having this in the settings page vs the peer connection page. Once the initial design is in place, it should be relatively simple to try both and see which one feels more natural.

**Comment by CaptainAchilles** [comment]
I think you're right @solmazabdi 

* `Settings` should be more of a global thing. Selecting mentor-ship/study should be a course-by-course basis because you may be comfortable mentoring one course but not another.
* However, the Users availability is more global thing in that it changes with you, regardless of your comfort with the courses. 

------
I think a lot of this will be decided by how we implement a multiple course offerings  (which we have spoken a bit about, but is out of scope at the moment).

**Comment by hkhosrav** [comment]
I believe all of the goals of this issue have been met; however, they have been moved over to the "Connect to Peers" page. Can we close the issue at this point?

**Comment by solmazabdi** [comment]
Hi,
In GitHub desk I cannot see improvement-availability#7 in current branch
tab. How can I fix it?
